launchcontainer/__init__.py

```diff
@@ -1 +1 @@
-from .launchcontainer import LaunchContainerXBlock
+
```

Comment:
Looks good to me, but you should also update the entry point to stop expecting the missing import:

```diff
     entry_points={
         'xblock.v1': [
-            'launchcontainer = launchcontainer:LaunchContainerXBlock',
+            'launchcontainer = launchcontainer.launchcontainer:LaunchContainerXBlock',
         ]
     },
```

Reply:
Good idea.